.github/workflows/kind-e2e-cosigned.yaml

```diff
@@ -0,0 +1,105 @@
+# Copyright 2021 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Policy Controller KinD E2E
```

Comment:
@k4leung4 I feel we can rename it to `Insecure registry KinD e2e`, as well as the filename of this gh action `kind-e2e-insecure-registry.yaml`. Or I am missing something.

Reply:
I also meant to rename the filename itself `.github/workflows/kind-e2e-cosigned.yaml` to `.github/workflows/kind-e2e-insecure-registry.yaml`, wdyt ?

Reply:
thanks for reminding me.